<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_ui</class>
 <widget class="QMainWindow" name="main_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1417</width>
    <height>1374</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1206</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>windowTitle</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="1" column="0" colspan="9">
     <layout class="QGridLayout" name="FftPlot"/>
    </item>
    <item row="9" column="4">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>FFT Zoom</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QCheckBox" name="cb_fft_ch2">
      <property name="text">
       <string>CH2</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QCheckBox" name="cb_fft_ch1">
      <property name="text">
       <string>CH1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="7">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="FftPlotToolbar">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
     </layout>
    </item>
    <item row="9" column="8" alignment="Qt::AlignRight">
     <widget class="QProgressBar" name="progressBar">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="format">
       <string>Done</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="9">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>263</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>400</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Furierreihe</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="4">
           <widget class="QPushButton" name="bn_cancle_measure">
            <property name="text">
             <string>Messung Abbrechen</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="bn_start_measure">
            <property name="text">
             <string>Messung Starten</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="5">
           <layout class="QHBoxLayout" name="SinusSignalPlot"/>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="cb_subsinus_visible">
            <property name="text">
             <string>Sub Sinus anzeigen</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>295</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Signal Generieren</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" alignment="Qt::AlignTop">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>275</width>
              <height>206</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>275</width>
              <height>206</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="2">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>V</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Anzahl Perioden</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="sb_abs">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>9999</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Faktor</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Anzahl Sampels</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="QPushButton" name="bn_calc_sinus">
               <property name="text">
                <string>Signal Generieren</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Offset</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="le_offset">
               <property name="inputMethodHints">
                <set>Qt::ImhDigitsOnly</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="le_factor">
               <property name="inputMethodHints">
                <set>Qt::ImhDigitsOnly</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="le_samples"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>368</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>368</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Sinus erzeugen</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="1">
           <widget class="QPushButton" name="bn_delete">
            <property name="text">
             <string>Löschen</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="bn_delete_all">
            <property name="text">
             <string>Alle Löschen</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="bn_add">
            <property name="text">
             <string>Hinzufügen</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QTableWidget" name="table_sinus">
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>110</number>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Frequenz [Hz]</string>
             </property>
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Amplitude [V]</string>
             </property>
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Phase [°]</string>
             </property>
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="9" column="6">
     <widget class="QSlider" name="hs_fft_slider">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
